README.md

```diff
@@ -6,6 +6,7 @@ Efficient, Expressive, Extensible HTML templates in JavaScript
 
 [![Build Status](https://travis-ci.org/Polymer/lit-html.svg?branch=master)](https://travis-ci.org/Polymer/lit-html)
 [![Published on npm](https://img.shields.io/npm/v/lit-html.svg)](https://www.npmjs.com/package/lit-html)
+[![Mentioned in Awesome lit-html>](https://awesome.re/mentioned-badge.svg)](https://github.com/web-padawan/awesome-lit-html)
```

Comment:
Is that `>` supposed to be there after `lit-html`?

Reply:
Oops, looks like it is copy-paste leftover. Fixed.